Formula/mold.rb

```diff
@@ -62,10 +62,13 @@ def install
 
     inreplace buildpath.glob("test/macho/*.sh"), "./ld64", bin/"ld64.mold", false
     inreplace buildpath.glob("test/elf/*.sh") do |s|
-      s.gsub!(%r{(`pwd`/)?mold-wrapper}, lib/"mold/mold-wrapper", false)
+      s.gsub!(%r{(\./|`pwd`/)?mold-wrapper}, lib/"mold/mold-wrapper", false)
       s.gsub!(%r{(\.|`pwd`)/mold}, bin/"mold", false)
-      s.gsub!(/-B[^\s]+/, "-B#{libexec}/mold", false)
+      s.gsub!(/-B(\.|`pwd`)/, "-B#{libexec}/mold", false)
     end
+    # Fix permission of test file.
+    # Remove on next release as fix is already merged upstream.
+    chmod "+x", buildpath/"test/elf/section-order.sh"
```

Comment:
Let's wait until the next release to use this then.

Reply:
Sure.